_posts/2018-07-19-IWD.md

```diff
@@ -0,0 +1,39 @@
+# International Women's Day (IWD)
+
```

Comment:
add some more content here, explain when this was, where it was, who organised (WTM)